File: stage1/problems/problem21/starter.java
import java.util.*;

public class starter {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        // --- Input reading ---
        String[] tokens = sc.nextLine().split(" ");
        int[] nums = new int[tokens.length];
        for(int i = 0; i < tokens.length; ++i) {
            nums[i] = Integer.parseInt(tokens[i]);
        }
        int k = sc.nextInt();

        // --- Solution ---
        // The following variables are available:
        // nums: int[]
        // k: int

        // TODO: Implement the solution
        
        sc.close();
    }
}

File: stage1/problems/problem21/starter.py
import sys

def solve():
  """
  Reads input from stdin, solves the problem, and prints the output.
  """
  try:
    # --- Input reading ---
    nums = list(map(int, sys.stdin.readline().split()))
    k = int(sys.stdin.readline())
    
    # --- Solution ---
    # The following variables are available:
    # nums: list of ints
    # k: int
    
    # TODO: Implement the solution
    
  except (ValueError, IndexError):
    pass

if __name__ == "__main__":
    solve()
